daprdocs/data/components/state_stores/generic.yaml

```
@@ -20,6 +20,17 @@
     etag: false
     ttl: true
     query: false
+- component: ClickHouse
+  link: setup-clickhouse
+  state: Alpha
+  version: v1
+  since: "1.15"
```

This is first released in v1.16 yes?
```suggestion
  since: "1.16"
```